custom/ewsghana/reports/__init__.py

```diff
@@ -119,6 +118,7 @@ def get_supply_points(self, location_id=None):
                 location_type__name__in=location_types,
                 parent=location
             )
+        locations.exclude(is_archived=True)
```

Comment:
I think you need to do `locations = locations.exclude(is_archived=True)` in order for this statement to do anything